abc.templates/base-workflows/contents/workflows/reusable-guardian-plan.yml

```diff
@@ -181,7 +172,7 @@ jobs:
 
   plan_success:
     if: |
-      always() && github.event_name == 'pull_request_target'
+      always()
```

Comment:
Looks like this will always (pun intended) succeed, so we can just remove the `if` statement.

Reply:
i think wee need to keep this but make it pull_request, this was added to support the mrege_group event i believe